projects/aca-content/src/lib/services/node-actions.service.ts

```diff
@@ -563,6 +563,47 @@ export class NodeActionsService {
     return null;
   }
 
+  getInfoDrawerIcon(node: Node): string {
```

Comment:
should be just 'getNodeIcon' ? why this service has a knowledge of the info drawer?

Reply:
Updated the name